docs/tutorial-model-and-deploy.md

```diff
@@ -0,0 +1,188 @@
+| DockerContainer_w1 | `http://opentosca.org/nodetypes` | The docker container hosting the QHAna frontend code. |
+| NGINX_latest-w1 | `http://opentosca.org/nodetypes` | The webserver delivering the QHAna frontend code. |
+| QHAnaUI_w1-wip1 | `https://ust-quantil.github.io/quantum/applications/nodetypes` | The QHAna frontend code. The node type is derived from `{http://opentosca.org/nodetypes}NGINX-Application_w1` |
+
+![topology with node templates and an open node template palette on the left](./images/topology-nodes-only-with-open-palette.png)
+
+Click on a *Node Template* to edit its *Properties* and to create new *Relationship Templates* by dragging one of the handles in the expanded Node Template to another node template.
+
+![topology with a node template selected and open properties sidebar on the right](./images/topology-node-selected.png)
+
+Connect all node templates in the topology with "HostedOn" relationship templates.
+Start with the QHAnaUI_w1-wip1 node template that is "HostedOn" the NGINX_latest-w1 node template that is "HostedOn" the DockerContainer_w1 node template that is "HostedOn" the DockerEngine_w1 node template.
+
+![topology with connected node templates](./images/topology-nodes-with-relationships.png)
+
+
+### 4. Set the Node Template Properties
+
+Use the menu above to show the properties in the node templates (click on "Properties" next to "Types").
+Open the container properties and fill in "ubuntu:20.04" in the "ImageID" property.
+Alternatively click on the node and edit the property in the sidebar on the right.
+Now fill in the other properties:
+
+| node template | property | value | explanation |
+|:--------------|:---------|:------|:------------|
+| DockerEngine_w1 | DockerEngineURL | `get_input: DockerEngineURL` | The URL to the docker engine that is used to create the containers. This tutorial uses the Docker-in-Docker (dind) container that is part of the OpenTOSCA docker-compose file. |
+| DockerContainer_w1 | ImageID | `ubuntu:20.04` | The base image to use. |
+| DockerContainer_w1 | ContainerPort | `80` | NGINX runs on port 80 so that port must be mapped outside. |
+| DockerContainer_w1 | Port | `get_input: Port` | The user input Port. Provided at deployment time by the end user. |
+| QHAnaUI_w1-wip1 | AppName | `qhana` | The app name is used as path to differentiate multiple NGINX apps hosted on the same NGINX server. |
+
+:information_source: The value `get_input: Port` is a special value that is replaced by a user input before deploying the service template.
+The name of the user input comes after the colon `get_input: <custom name here>`.
+All properties with the same user input name in the same service template will **share** the value of the user input.
+
+![topology with properties shown in node templates](./images/topology-with-properties.png)
+
+
+### 5. Add the Deployment Artifact
```

Comment:
I would skip this step and alreay add it in the Node Type Implementation of the QHAna UI.

Reply:
I think it is better to leave it in the tutorial, as it was not trivial for me to do and is something that will be useful to know for everyone who wants to model applications with TOSCA.